src/projects.md

```diff
@@ -89,6 +89,7 @@ of Medicine to meet the needs of their own research. Below is a partial list of
   + <a href="http://pblab.tau.ac.il/en/" target="_blank">Pablo Blinder Lab</a> (<a href="https://github.com/PBLab" target="_blank">GitHub</a>)
 + University of Bonn
   + <a href="https://rose-group.ieecr-bonn.de" target="_blank">Tobias Rose Lab</a>
+  + <a href="https://https://www.ukbonn.de/en/epileptology/workgroups/mormann-workgroup-cognitive-und-clinical-neurophysiology/">Mormann Workgroup</a>
```

Comment:
Please also update the the `<!-- numbers -->` in headings above to update our running count.

Reply:
TBD in subsequent PR's.